tests/fixtures/locator.spdx.json

```diff
@@ -0,0 +1,162 @@
+{
+    "spdxVersion": "SPDX-2.2",
+    "documentNamespace": "https://localmaven",
+    "creationInfo": {
+        "creators": [
+            "Tool: github.com/phylum-dev/cli"
+        ],
```

Comment:
I see now... We're testing against an SPDX file that was created by [our own API][1]...

[1]: https://api.staging.phylum.io/api/v0/swagger/index.html#/Jobs/jobs_get_job_sbom

That makes some sense because we should be able to parse these files. But there are a few big problems with that:

1. Out API currently uses `"referenceType"` values that don't exist in the spec
2. Our API is currently working to switch to PURL, so these locators will be phased out.
3. Even for reference types that **are** defined in the spec, our API seems to deviate from the spec (e.g., `nuget`)

Can we use some other tool to generate our test files?

Reply:
I was having trouble finding anything not using PURL. The spec only has references for a few ecosystems. It seems abandoned in favor of PURL.

Reply:
We can probably still support it given it's in the spec. I used https://github.com/spdx/tools-python to convert back and forth to help finalize this file (knowing need to update the API output).